tcs34725.py

```diff
@@ -123,6 +123,23 @@ def _temperature_and_lux(self, data):
         cct = 449.0 * n**3 + 3525.0 * n**2 + 6823.3 * n + 5520.33
         return cct, y
 
+    def html_rgb(self):
+        r, g, b, c = self.read(raw=True)
+        red = int((r/c) * 256)
+        green = int((g/c) * 256)
+        blue = int((b/c) * 256)
```

Comment:
Instead of calling the `self.read` yourself, it would be better to just provide a conversion function that takes a 4-tuple as the parameter and returns the html rgb, or even better, returns a 3-tuple.

Also, documentation is missing.